src/Support/Support/help/Update-AzSupportTicket.md

```diff
@@ -38,13 +38,13 @@ Update-AzSupportTicket -InputObject <PSSupportTicket> [-Severity <Severity>] [-C
 
 ### UpdateByInputObjectWithContactObjectParameterSet
 ```
-Update-AzSupportTicket -InputObject <PSSupportTicket> [-Severity <Severity>]
+Update-AzSupportTicket -InputObject <PSSupportTicket> [-Severity <Severity>] [-Status <Status>]
  -CustomerContactDetail <PSContactProfile> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
  [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-Use this cmdlet to update a support ticket's severity level or customer contact details. Note that updating a support ticket's severity level is not allowed when the ticket is assigned to a support engineer. If you wish to update the severity level after ticket assignment, contact the support engineer by sending a communication on the ticket.
+Use this cmdlet to update a support ticket's severity level, status or customer contact details. Note that updating a support ticket's severity level or status is not allowed when the ticket is assigned to a support engineer. If you wish to update the severity level or status after ticket assignment, contact the support engineer by sending a communication on the ticket.
```

Comment:
"If you wish to update the severity level or status when an engineer is actively working on your ticket, send a communication on the ticket to the support engineer and request for the severity/status update"